osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/infra/messaging/processors/KeysRequestMessageProcessor.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2015 Smart Society Services B.V.
```

Comment:
This is not the correct copyright statement for newly added code.

Reply:
Changed copyright header